src/lib/permissions/proposals/proposalFlowFlags.ts

```diff
@@ -102,8 +105,8 @@ function withDepsReviewedProposal({ computeProposalPermissions }: GetFlagFilterD
       userId
     });
 
-    if (permissions.create_vote) {
-      flags.addPermissions(['vote_active']);
+    if (permissions.create_vote || (await isProposalReviewer({ proposal, userId })) === true) {
```

Comment:
yuck, async in the middle of conditional logic... but i see this pattern already exists :(